src/common/components/ReviewModal/ReviewModal.tsx

```diff
@@ -0,0 +1,87 @@
+import { Modal } from "@/common/components/Modal";
+import { type Review } from "@/common/components/ReviewItem";
+import { ProfileSchool } from "@/common/components/ReviewItem/ProfileSchool";
+import { ReviewBody } from "@/common/components/ReviewItem/ReviewBody";
+import ReviewHeader from "@/common/components/ReviewItem/ReviewHeader";
+import { reviewModalTheme } from "@/common/components/ReviewModal/ReviewModal.theme";
+import { getHumanReadableTimestampDelta } from "@/common/functions";
+import { Icon } from "@iconify-icon/react/dist/iconify.mjs";
```

Comment:
nit(non-blocking): try avoid iconify and use CustomIcons as much as possible since there are [issues with using the lib](https://github.com/AfterClass-io/afterclass.io-v2/pull/59#discussion_r1460368031) and we are [migrating away from it](https://github.com/AfterClass-io/afterclass.io-v2/issues/61)

for the like icon, we have it [`@/common/components/CustomIcon/ThumbUpFilledIcon`](https://github.com/AfterClass-io/afterclass.io-v2/blob/main/src/common/components/CustomIcon/ThumbUpFilledIcon.tsx)

svgs can be exported from figma and added directly to `CustomIcon`

Reply:
fixed https://github.com/AfterClass-io/afterclass.io-v2/pull/104/commits/79fb3d79570fd8a8455ef4e70dcc687af5bbf8b4